### PR TITLE
Feature/scroll indicator desktop

### PR DESCRIPTION
**Pull request by mfbarrera**
The scroll indicator for desktop is ready to review.

the Scroll indicator for desktop with its copy shows and works as scroll button

https://trello.com/c/KUxG3ffU/6-desktop-indicador-de-scroll-3

https://app.zeplin.io/project/5f00e6fda64d6e11dca0753a/screen/5f00e7c421dfb3189414cceb

image

### CODE REVIEW

File: src/scss/components/_scroll-indicator.scss [hunk 1]
```diff
@@ -1,37 +1,60 @@
...
     border: none;
     display: flex;
-    height: 40px;
-    margin:auto;
+    flex-direction: column;
+    margin: auto;
     width: 40px;
   }
 
-  &__button:focus {
```
Comment (caritonaranjo) [inline src/scss/components/_scroll-indicator.scss:23]: why eliminated this focus online: none?

**Review by caritonaranjo — COMMENTED** [review]
excellent work Fer!!!
one question: the arrow and the text are one button, will not be better if the border of that button were invisible, so the user doesńt notice that?

File: src/scss/components/_scroll-indicator.scss (hunk 1 above)
Comment (jusk02) [reply src/scss/components/_scroll-indicator.scss:23]: @caritonaranjo not having a button outline may represent an accessibility problem. Maybe some custom styles can be set to ensure it integrates well with de design.

**Comment by jusk02** [comment]
@mfbarrera please review the suggested PR composition
https://docs.google.com/document/d/1sXwXD8hiT0xbTuqzo2ljxUYsQcR5EQnHkCQ5SmUcEzE/edit

File: src/scss/components/_scroll-indicator.scss (hunk 1 above)
Comment (mfbarrera) [reply src/scss/components/_scroll-indicator.scss:23]: It is exactly the reason @jusk02 describe above. Accessibility mention the focus should be visible.

**Review by caritonaranjo — APPROVED** [review]

File: src/scss/components/_scroll-indicator.scss [hunk 2]
```diff
@@ -1,37 +1,60 @@
...
+      margin-bottom: 100px;
       text-align: center;
     }
+
+    &__image {
+      background-color: transparent;
+      width: 40px;
+    }
+
+    &__bounce {
```
Comment (jsebax) [inline src/scss/components/_scroll-indicator.scss]: This could be a modifier of the `__button`, as it is part of the component state or behavior

**Comment by jsebax** [comment]
Add the property `cursor: pointer;` to the buttons of the page. You could add this as a global style in the `_reset.scss` file.

File: src/js/components/footer.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ const footerComp = (data) => {
   const socialText = newText('p', data.socialText, ['footer__social-text']);
   const socialIcons = createIcons(data.socialLinks);
   linkLogo.appendChild(logoFooter);
-  const footer = newContainer('footer', [linkLogo, items, socialText, ...socialIcons], ['footer']);
+  const footer = newContainer('footer', [linkLogo, items, socialText, ...socialIcons], ['footer','footer__desktop']);
```
Comment (jsebax) [inline src/js/components/footer.js]: What are you modifying this file? It's not part of the your component

File: src/scss/layout/_footer.scss [hunk 4]
```diff
@@ -35,9 +35,17 @@
 
     &-links {
       display: inline-block;
-        &:not(:last-of-type) {
+      &:not(:last-of-type) {
```
Comment (jsebax) [inline src/scss/layout/_footer.scss]: Leave a space before this line

File: src/scss/layout/_footer.scss [hunk 5]
```diff
@@ -35,9 +35,17 @@
...
     }
   }
+
+  @include viewport-desktop() {
+      &__desktop {
+        padding: 111px 106px 111px 124px;
+        display: flex;
+      }
+  }
+
```
Comment (jsebax) [inline src/scss/layout/_footer.scss]: Remove this line

**Review by jsebax — APPROVED** [review]

**Pull request merged**